src/main/bin/ccm-fetch

```diff
@@ -299,10 +302,22 @@ if (!defined($errno)) {
 } elsif ($errno == $ERROR) {
     $this_app->error("Unable to fetch profile");
     exit(1);
-} elsif ($errno == CHANGED) {
-    $this_app->info("Profile updated");
 } else {
-    $this_app->verbose("Fetched profile was not new");
+    # Init cachemanager
+    my $cmgr = EDG::WP4::CCM::CacheManager->new($fetch->{CACHE_ROOT}, $fetch->{_CCFG});
+    # get Configuration instance (anonymous, latest CID)
+    my $cfg = $cmgr->getConfiguration();
+    # get name and CID
+    my $msg = "latest CID ".$cfg->getConfigurationId();
+
+    my $name = $cfg->getName();
+    $msg .= " name $name" if defined($name);
+
+    if ($errno == CHANGED) {
+        $this_app->info("Profile updated, new $msg");
+    } else {
+        $this_app->info("Fetched profile was not new, $msg");
```

Comment:
my view is that it doesn't really matter if the profile is new or not, what really matter is latest the CID/name.

but i'll change it, altough i'd always print the name, the CID itself is interesting but not that much. the name should be much more relevant.